test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -521,6 +521,69 @@ def test_res_boolean_invalid_input(self):
         a = {'job_state': 'Q', 'comment': job_comment}
         self.server.expect(JOB, a, id=jid, attrop=PTL_AND)
 
+    def test_res_timeout(self):
+        """
+        Test server_dyn_res script timeouts after 30 seconds
+        """
+
+        # Create a resource of type boolean
+        resname = ["foo"]
+        restype = ["boolean"]
+
+        # Prep for server_dyn_resource script
+        resval = ["sleep 60\necho true"]
+
+        filenames = self.setup_dyn_res(resname, restype, resval)
+
+        # Submit job
+        a = {'Resource_List.foo': '"true"'}
```

Comment:
Why do you need the extra quotes?

Reply:
Copy-pasted it from another test, I've removed it